drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/aggregated/combine_enrollments.rb

```diff
@@ -187,7 +187,11 @@ def rebuild_warehouse_data
             ProjectID: combined_project_ids,
           ),
         ).distinct.pluck(:id)
-      GrdaWarehouse::Tasks::SanityCheckServiceHistory.new(client_ids: dest_clients).run!
+
+      # Mark all of these clients as needing a full rebuild so we don't end up with thousands of service history services
+      GrdaWarehouse::Hud::Client.where(id: dest_clients).
+        each(&:invalidate_service_history)
+      GrdaWarehouse::Tasks::ServiceHistory::Add.new.run!
```

Comment:
Is this redundant with `post_process` in the importer?

Reply:
Possibly, is `GrdaWarehouse::Tasks::ServiceHistory::Add.new.run!` equivalent to `GrdaWarehouse::Tasks::ServiceHistory::Enrollment.queue_batch_process_unprocessed!`